coderd/database/migrations/fix_migration_numbers.sh

```diff
@@ -11,7 +11,7 @@ list_migrations() {
 main() {
 	cd "${SCRIPT_DIR}"
 
-	origin=$(git remote -v | grep "github.com[:/]coder/coder.*(fetch)" | cut -f1)
+	origin=$(git remote -v | grep "github.com[:/]*coder/coder.*(fetch)" | cut -f1)
```

Comment:
```bash
$ git remote -v | grep "github.com[:/]coder/coder.*(fetch)" | cut -f1

$ $ git remote -v | grep "github.com[:/]*coder/coder.*(fetch)" | cut -f1
origin
```

In fact, hilariously, the former will actually cause the whole script to bork because of `pipefail`, since `grep` exits 1.